src/main/java/seedu/entities/Food.java

```diff
@@ -0,0 +1,134 @@
+package seedu.entities;
+
+public abstract class Food {
+    protected int id;
+    protected String name;
+    protected String storeName;
+    protected int storeNumber;
+    protected float calories;
+    protected float protein;
+    protected float totalFat;
+    protected float saturatedFat;
+    protected float dietaryFibre;
+    protected float carbohydrates;
+    protected float sugar;
+    protected float sodium;
```

Comment:
Consider use private modifier since it is only accessed and modified within the class